src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/integration_tests/tests/web/WorkflowRunTests.kt

```diff
@@ -454,6 +454,29 @@ class WorkflowRunTests : IntegrationTest()
         assertThat(errors[0]["code"].asText()).isEqualTo("bad-request")
     }
 
+    @Test
+    fun `can get workflow summary`()
+    {
+        val ref = getGitBranchCommit("master")
+        val sessionCookie = webRequestHelper.webLoginWithMontagu(runReportsPerm)
+        val response = webRequestHelper.requestWithSessionCookie(
+                "/workflows/summary",
+                sessionCookie,
+                ContentTypes.json,
+                HttpMethod.post,
+                """{"ref": "$ref", "reports": [{"name": "global", "params": {}}]}"""
+        )
+
+        assertSuccessful(response)
+        assertJsonContentType(response)
+        val json = JsonLoader.fromString(response.text)
+        val data =  json["data"].toString()
+        assertThat("missing_dependencies" in data).isEqualTo(true)
+        // global should appear twice: once under reports and once under missing_dependencies
+        // therefore splitting the string in two places should produce three fragments
+        assertThat(data.split("global").count()).isEqualTo(3)
```

Comment:
There are probably more elegant ways to check the response but I was struggling to get anything that involved casting to an ArrayNode to work.